src/module/actor/actions/simple.ts

```diff
@@ -73,10 +73,11 @@ class SimpleActionVariant extends BaseActionVariant {
         });
         const results: SimpleActionResult[] = [];
         for (const actor of actors) {
-            const message = await ChatMessage.create({
+            const data = {
                 flavor,
                 speaker: ChatMessage.getSpeaker({ actor }),
-            });
+            };
+            const message = options.message?.create ? await ChatMessage.create(data) : new ChatMessage(data);
```

Comment:
defaults to false?

Reply:
That's what you get for making last second changes. Good catch!